src/adapters/postgres/user-adapter.ts

```
@@ -169,11 +169,29 @@ export class PostgresUserService implements IServicelocator {
         ? programName.charAt(0).toUpperCase() + programName.slice(1)
         : 'Learner Account';
 
+      // Check if role is sent from request body
+      const requestBody = request?.body || {};
+      const roleFromBody = requestBody.role;
+
+      let notificationKey;
+
+      if (roleFromBody) {
+        // If role is sent from body, use role-specific notification key
+        if (roleFromBody === 'Regional Admin') {
```

ESLint

[error] Replace `'Regional·Admin'` with `"Regional·Admin"`

(prettier/prettier)